beets/library.py

```diff
@@ -1466,6 +1466,34 @@ def tmpl_aunique(self, keys=None, disam=None):
         self.lib._memotable[memokey] = res
         return res
 
+    @staticmethod
+    def tmpl_first(s, count=1, skip=0, sep=u'; ', join_str=u'; '):
+        """ Gets the item(s) from x to y in a string separated by something
+        and join then with something
+
+        :param s: the string
+        :param count: The number of items included
+        :param skip: The number of items skipped
+        :param sep: the separator. Usually is '; ' (default) or '/ '
+        :param join_str: the string which will join the items, default '; '.
+        """
+        skip = int(skip)
+        count = skip + int(count)
+        return join_str.join(s.split(sep)[skip:count])
+
+    def tmpl_ifdef(self, field, trueval=u'', falseval=u' '):
```

Comment:
this was one workaround because of the default path regex replacements. I think it was `'[\\/]': _`.
The path was replacing '%ifdef{something}/' with '/_/' instead of '/' if this function was in the beginning of the path.